docs/esp32/quickref.rst

```diff
@@ -507,6 +507,21 @@ The DHT driver is implemented in software and works on all pins::
     d.temperature() # eg. 23.6 (°C)
     d.humidity()    # eg. 41.3 (% RH)
 
+SD card
```

Comment:
as with WDT, since this is in the machine module, would be better grouped with other machine things (but at the end of the group because it's more advanced)

Reply:
@dpgeorge Done. Moved higher up.